bindata/assets/kube-descheduler/softtainterclusterrole.yaml

```diff
@@ -0,0 +1,52 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: openshift-descheduler-softtainter
+rules:
+- apiGroups:
+  - "events.k8s.io"
+  resources:
+  - "events"
+  verbs:
+  - "get"
+  - "watch"
+  - "list"
+  - "create"
+  - "update"
+  - "patch"
+  - "delete"
+- apiGroups:
+  - ""
+  resources:
+  - "nodes"
+  verbs:
+  - "get"
+  - "watch"
+  - "list"
+  - "patch"
+  - "update"
+- apiGroups:
+  - "coordination.k8s.io"
+  resources:
+  - "leases"
+  verbs:
+  - "create"
+- apiGroups:
+  - "coordination.k8s.io"
+  resources:
+  - "leases"
+  resourceNames:
+  - "soft-tainter-lock"
+  verbs:
+  - "get"
+  - "patch"
+  - "update"
+  - "delete"
+- apiGroups:
+    - "operator.openshift.io"
+  resources:
+    - "kubedeschedulers"
+  verbs:
+    - "get"
+    - "watch"
+    - "list"
```

Comment:
Uber nit: can we fix the indentation of these arrays ?

Reply:
done